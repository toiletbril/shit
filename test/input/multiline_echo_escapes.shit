echo \
is_escaped\* \
word \
\
does_not_exist*
