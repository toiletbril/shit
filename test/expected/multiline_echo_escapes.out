[CompoundList]
  [CompoundListCondition, None]
    [SimpleCommand "echo is_escaped* word does_not_exist*"]
[Escape Map
  00000100 00000000 10001000 00011000 
]
shit: 5:1: Error: Could not expand path: No expansions found for 'does_not_exist*'.
     5 |  does_not_exist*
       |  ^~~~~~~~~~~~~~~ here.
